### PR TITLE
Pull Request for #19: Confidence interval for SK distance

### PR DESCRIPTION
**Pull request by rcalvo12**
In this issue, we added the function `get_sk_ci()` to the `helpers.py` along with documentation to the `BootstrapReport.lyx` file. We also added a unit test.

### CODE REVIEW

**Comment by rcalvo12** [comment]
@jmshapir, I was unable to add @isandrews as a reviewer. I am not sure he is officially a contributor to this repository?

File: src/BootstrapReport/helpers.py [hunk 1]
```diff
@@ -138,6 +138,21 @@ def get_sk_dist(rep, normal, sep = False):
     else:
         return -neg_dist + pos_dist
 
+def get_sk_ci(num_replicates, neg_dist, pos_dist, alpha = 0.05):
+    '''calculates confidence interval for sk distance
+    :param num_replicates:number of replicates
```
Comment (jmshapir) [inline src/BootstrapReport/helpers.py:143]:
```suggestion
    :param num_replicates: number of replicates
```

File: src/BootstrapReport/main.py [hunk 2]
```diff
@@ -25,7 +25,9 @@ def __init__(self, estimate, se, replicates):
         self.best_bandwidth_value = np.NaN
         self.bias_value = None
         self.crossings = None
-        self.sk_dist = helpers.get_sk_dist(self.replicates, norm(loc = estimate, scale = se), sep = False)
+        self.sk_neg, self.sk_pos = helpers.get_sk_dist(self.replicates, norm(loc = estimate, scale = se), sep = True)
```
Comment (jmshapir) [inline src/BootstrapReport/main.py:28]:
```suggestion
        self.sk_neg, self.sk_pos = helpers.get_sk_dist(self.replicates, norm(loc = self.estimate, scale = self.se), sep = True)
```

**Review by jmshapir — APPROVED** [review]
@rcalvo12 thanks!

I have only shallow comments so I'm approving.

@MosesStewart it would be great if you can confirm this update meets the requirements in https://github.com/JMSLab/BootstrapReport/issues/19#issue-2002786349.

@isandrews I've invited you to the repository. Once you accept the invite I can add you as reviewer here. It would be great if you can take a quick look at the new method in case something looks off to you. If all looks ok feel free to approve.

Thanks all!

**Comment by isandrews** [comment]
Thanks very much @rcalvo12 - this looks very good to me!  One small request: for the upper bound, after adding the two components could we again take the min with 1?  I think this should rarely matter, but there may be cases where the DWK interval is very wide and we get a value above 1 if we omit that additional min.

**Comment by jmshapir** [comment]
@isandrews thanks!

> Thanks very much @rcalvo12 - this looks very good to me! One small request: for the upper bound, after adding the two components could we again take the min with 1? I think this should rarely matter, but there may be cases where the DWK interval is very wide and we get a value above 1 if we omit that additional min.

In eb19b13 I modified the LyX doc to reflect that proposal. Let me know if it looks off to you.

@rcalvo12 please let us know if it doesn't make sense; otherwise feel free to implement when ready!

**Review by MosesStewart — APPROVED** [review]
The code changes match the equation in https://github.com/JMSLab/BootstrapReport/issues/19#issuecomment-1819623402 !

**Comment by jmshapir** [comment]
@rcalvo12 thanks!

@isandrews please approve the pull if all looks good to you now, thank you!

**Comment by rcalvo12** [comment]
Summary in https://github.com/JMSLab/BootstrapReport/issues/19#issuecomment-1834337200.

**Pull request merged**